client/src/pages/support.js

```diff
@@ -0,0 +1,118 @@
+      <Spacer />
+      <Grid className='container'>
+        <Row>
+          <Col
+            className='questions'
+            md={6}
+            mdOffset={3}
+            sm={10}
+            smOffset={1}
+            xs={12}
+            >
+            <h2 className='text-center'>Common Technical Support Questions</h2>
+            <hr />
+            <h4 id='faq_progress'>
+              I just signed into my account and I don't see any of my past
+              progress.
+            </h4>
+            <p>
+              You have created a duplicate account.{' '}
+              <Link to='/settings'>
+                Sign out of your account
+              </Link>{' '}
+              and try signing in using a different service (Google, GitHub,
+              Facebook) that you may have used to in the past. Or try signing in
+              using an email address you may have used on freeCodeCamp in the
+              past.
+            </p>
+            <h4 id='faq_donation'>
+              I set up a monthly donation, but I need to update or cancel the
+              monthly recurrence. How can I do this?
+            </h4>
+            <p>
+              We are working on a dashboard for you to be able to update or
+              cancel your donations right from your settings page. In the
+              meantime, if you need to update or cancel your monthly donation,
+              forward an invoice you received in your email to{' '}
+              <Link to='mailto:team@freecodecamp.org'>team@freecodecamp.org</Link>{' '}
+              and tell us what you'd like us to do, and we'll take care of it
+              for you in our database.
+            </p>
+            <h4 id='faq_solutions'>How can I view my past solutions?</h4>
+            <p>
+              We have archived the millions of solutions from prior to June
+              2018, and are working on a sustainable way to host them and future
+              solutions.
+            </p>
+            <p>
+              You can meanwhile use the download my solution button to view and
+              copy the solution after you complete a challenge, from the local
+              copy on your browser.
+            </p>
+            <h4 id='faq_streak'>My streak on my portfolio is inaccurate.</h4>
+            <p>
+              We are working on fixing a number of issues with the streaks on
+              freeCodeCamp. Thanks for your patience.
+            </p>
+            <h4>
+              When I go to{' '}
+              <Link to='/learn'>
+                Learning Curriculum
+              </Link>{' '}
+              the challenges are completely blank.
+            </h4>
+            <p>
+              Do a hard refresh of the website by pressing control+shift+r in
+              Windows or command+shift+r on Mac/Linux. If that doesn't work, you
+              may need to clear your cookies. Here is{' '}
+              <Link to='/forum/t/205075'>
```

Comment:
We can't use `/forum` links without `external={true}` because of Gatsby don't know about them.